src/main/scala/cromwell/engine/backend/jes/authentication/JesAuthentication.scala

```diff
@@ -1,30 +1,33 @@
 package cromwell.engine.backend.jes.authentication
 
 import cromwell.engine.backend.jes._
-import cromwell.util.google.{GoogleCloudStorage, GoogleCredentialFactory}
+import cromwell.engine.io.gcs.GoogleCloudStorage
+import cromwell.util.google.GoogleCredentialFactory
 import spray.json.JsObject
 
 /**
  * Trait for JesConnection
  */
 trait JesConnection {
-  def jesCromwellConnection: JesInterface
+  def jesCromwellInterface: JesInterface
 
   /**
     * This method should try its best to provide a GCS connection setup with the user's credentials.
     * In the case where it's not able to provide such a method, a default one can be provided instead.
     */
-  def jesUserConnection(backendCall: JesBackendCall): JesBackend.IOInterface
+  def jesUserConnection(backendCall: JesBackendCall): GoogleCloudStorage
 }
 
 object ProductionJesConnection {
   import ProductionJesConfiguration._
 
-  // Only one instance of jesCromwellConnection is needed. It uses whichever authScheme has been set in the configuration.
-  lazy val jesCromwellConnection: JesInterface = {
-    val cromwellCredentials = GoogleCredentialFactory.fromAuthScheme
-    val gcsInterface = GcsFactory(jesConf.applicationName, cromwellCredentials)
-    val genomicsInterface = GenomicsFactory(jesConf.applicationName, jesConf.endpointUrl, cromwellCredentials)
+  // Only one instance of jesCromwellInterface is needed. It uses whichever authScheme has been set in the configuration.
+  lazy val jesCromwellInterface: JesInterface = {
+    val cromwellCredentials = GoogleCredentialFactory.fromCromwellAuthScheme
+    // .get to fail now, as we can't run on Jes without a Cromwell authenticated GCS interface
+    val gcsInterface = GoogleCloudStorage.cromwellAuthenticated.get
```

Comment:
Oh actually this one is a `Try` which encapsulates the potential configuration parsing errors leading to the failure (same as your other comment).
However the one line 70 for the `jesUserConnection` is an Option and definitely could use a better exception.